app/services/embedded_page_service.rb

```diff
@@ -0,0 +1,90 @@
+# Processes requests for pages embedded in iframes
+
+class EmbeddedPageService
+  attr_reader :use_embedded_layout
```

Comment:
Now here is a disadvantage of the attr_reader. This method should be called `use_embedded_layout?`, right?

Reply:
let's make that a regular method if we want the question mark then. It's the only way if we want it.